api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetupMERTxNew2019Report.java

```diff
@@ -124,9 +126,11 @@ public ReportDefinition constructReportDefinition() {
         CohortIndicatorDataSetDefinition dsd = new CohortIndicatorDataSetDefinition();
 
         dsd.setParameters(getParameters());
-        rd.addDataSetDefinition("TX_NEW", Mapped.mapStraightThrough(dsd));
+        rd.addDataSetDefinition("TX", Mapped.mapStraightThrough(dsd));
+        rd.addDataSetDefinition("S", Mapped.mapStraightThrough(settings()));
+        rd.addDataSetDefinition("aijar", Mapped.mapStraightThrough(getUgandaEMRVersion()));
 
-        CohortDefinitionDimension ageDimension = commonDimensionLibrary.getTxNewAgeGenderGroup();
+        CohortDefinitionDimension ageDimension = commonDimensionLibrary.getNewTxCurrAgeGenderGroup();
```

Comment:
@dbaluku or maybe rename it to getFinerAgeWith55And65Ranges() so that it is clear to the reader (I am not sure of the ranges added so they may need to be corrected)

Reply:
Done